pages/agent/v3/securing.md

```diff
@@ -149,7 +149,17 @@ exit 1
 
 ## Forcing clean checkouts
 
-By default Buildkite will reuse (after cleaning) a previous checkout. This may not be safe if building commits from untrusted sources (for example, 3rd party pull requests). To force a clean checkout every time, set `BUILDKITE_CLEAN_CHECKOUT=true` in the environment.
+By default, Buildkite will reuse (after cleaning) a previous checkout. This may be unsafe if building commits from untrusted sources (for example, third-party pull requests). To force a clean checkout every time, set BUILDKITE_CLEAN_CHECKOUT=true in the environment. The following example shows how to enforce a clean checkout at the step level:
+
+```yaml
+steps:
+- label: ":github: Clean Checkout"
```

Comment:
@dannymidnight, any idea why the emoji is rendering like this in [the preview](https://2261--bk-docs-preview.netlify.app/docs/agent/v3/securing#forcing-clean-checkouts):
![Screenshot 2023-07-21 at 3 20 20 pm](https://github.com/buildkite/docs/assets/119824349/ec39f38f-0cfe-41d9-875a-d40a1e9f32ed)